getstarted.md

```diff
@@ -3,72 +3,203 @@ layout: page
 title: Get Started
 ---
 
-React Hot Loader is a plugin for Webpack that allows instantaneous live refresh without losing state while editing React components.
+React Hot Loader is a plugin that allows instantaneous live refresh without losing state while editing React components.
 
-If you use Browserify, RequireJS or another JavaScript bundler, you need to switch to Webpack first. Webpack supports all popular module formats. If you know a good guide on migrating to Webpack, <a href="https://github.com/gaearon/react-hot-loader/issues/new" target="_blank">let me know</a> and Iʼll link to it.
+It works with Webpack and other bundlers that support both Hot Module Replacement (HMR) and Babel plugins.
 
-This tutorial assumes that you already have a working Webpack configuration and `WebpackDevServer` compiles and serves your code. If youʼd rather play with a ready-made example, try <a href="https://github.com/gaearon/react-hot-boilerplate" target="_blank">react-hot-boilerplate</a>.
+## Boilerplate Example
```

Comment:
Agree.  If you're already working on this, feel free to just change it.  Don't forget to pull in the changes from the other PR first though.